vite.config.ts

```diff
@@ -63,6 +63,15 @@ export default defineConfig(({ mode }) => {
   // Allow hot reloading of json files in public folder when in development
   if (env.NODE_ENV === 'development') plugins.push(jsonHMR());
 
+  if (env.VITE_APP_INCLUDE_CODECOV === 'true')
```

Comment:
```suggestion
  // Allow codecov bundle analysis
  if (env.VITE_APP_INCLUDE_CODECOV_PLUGIN === 'true')
```
I think the name is more explicit that its a plugin and the comment matches those above.